src/views/instructor/training/EditSession.vue

```diff
@@ -108,7 +108,7 @@
 					</div>
 					<div class="row row_no_margin">
 						<div class="input-field col s12 submit_buttons">
-							<button type="button" v-if="step === 3" class="btn right" @click="submitForm">Finalize</button>
+							<button type="button" v-if="step === 3" class="btn right" @click="submitForm(); submitTraining(); ">Send to VATUSA</button>
```

Comment:
Can we make one function that calls submitForm and submit training? Kinda confusing to have the button doing two actions.